### ISSUE TITLE
[FR] Raise on suspicious alt text

### ISSUE THREAD

**Issue by r-leyshon** [rstudio/rmarkdown#2372]
Documents rendered with inappropriate alt text for images and charts result in accessibility issues for screen reader users. The [approach taken with `accessrmd`](https://github.com/datasciencecampus/accessrmd/blob/main/R/sus_alt.R). It could be best to raise warnings on knit if any image alt text were equal to the following:

- [ ] no alt attribute value
- [ ] 'nbsp'
- [ ] 'spacer'
- [ ]  alt attribute is the same as src attribute value

Additionally, English, German & Korean language values have associated character limits for alt text. Please [see the `accessrmd` implementation](https://github.com/datasciencecampus/accessrmd/blob/main/R/find_alt_lim.R).

**Further reading**
* [W3C WCAG2.0 alt text guidance](https://www.w3.org/WAI/GL/WCAG20/tests/test3.html)
* [W3C WCAG2.0 alt text placeholder guidance](https://www.w3.org/WAI/GL/WCAG20/tests/test6.html)
